internal/gitlab/gitlab.go

```diff
@@ -1900,3 +1906,18 @@ func CreateMergeRequestCommitDiscussion(project string, mrID int, sha string, ne
 	note := discussion.Notes[0]
 	return fmt.Sprintf("%s/merge_requests/%d#note_%d", p.WebURL, note.NoteableIID, note.ID), nil
 }
+
+// hasNextPage is a pagination helper to handle the case where the API
+// response comes without the HTTP X-Total(-Page) header. Reference:
+// https://docs.gitlab.com/ee/user/gitlab_com/index.html#pagination-response-headers
+func hasNextPage(resp *gitlab.Response) (int, bool) {
+	if resp.TotalPages != 0 {
+		if resp.CurrentPage >= resp.TotalPages {
+			return 0, false
+		}
```

Comment:
I don't see the point of keeping this code, since we are returning NextPage anyway.

Reply:
That's a good point. Considering that only `X-Total` and `X-Total-Pages` might not be present in the header, I think we can just not use it at all. I'll remove it and send a new version.